tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -2508,6 +2511,8 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
     rinfo_.push_back({csinfo_.mul,
                       mkl_op_registry::GetMklOpName(csinfo_.mul),
                       CopyAttrsDataType, AlwaysRewrite});
+    rinfo_.push_back({csinfo_.pad_with_conv2d, csinfo_.mkl_pad_with_conv2d,
```

Comment:
Actually I used csinfo._pad_with_conv2d for __MklDummyPadWithConv2D, and csinfo._mkl_pad_with_conv2d for _MklPadWithConv2D. Here, we have a dummy Mkl node (__MklDummyPadWithConv2D) when we merge pad and conv2d nodes. Then this dummy Mkl node is replaced by actual Mkl merged node _MklPadWithConv2D. This dummy node is needed only for merged node. Unmerged node does not need dummy node. If we try to use mkl_op_registry::GetMklOpName(csinfo_.mkl_pad_with_conv2d), then it will produce node name as _Mkl__MklPadWithConv2D, which is not intended. We here follow the same convention as the merging of conv2d+Bias as seen: https://github.com/Intel-tensorflow/tensorflow/blob/dd63093a599081accfe2a2d2ca8c029d413a15d7/tensorflow/core/graph/mkl_layout_pass.cc#L2475

Reply:
Got it. Thank you for the explanation!